ci/binary_size_treemap.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+#
+# Copyright 2013 The Flutter Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+# Run a tool that generates a treemap showing the contribution of each
+# component to the size of a binary.
+#
+# Usage:
+#   binary_size_treemap.sh [binary_path] [output_dir]
+
+set -e
+
+INPUT_PATH="$(cd $(dirname "$1"); pwd -P)/$(basename "$1")"
+DEST_DIR="$(cd $(dirname "$2"); pwd -P)/$(basename "$2")"
+CI_DIRECTORY=$(cd $(dirname "${BASH_SOURCE[0]}"); pwd -P)
+ENGINE_BUILDROOT=$(cd "$CI_DIRECTORY/../.."; pwd -P)
+
+echo $INPUT_PATH
+echo $DEST_DIR
+
+if [ "$(uname)" == "Darwin" ]; then
+  NDK_PLATFORM="darwin-x86_64"
+else
+  NDK_PLATFORM="linux-x86_64"
+fi
+ADDR2LINE="third_party/android_tools/ndk/toolchains/aarch64-linux-android-4.9/prebuilt/$NDK_PLATFORM/bin/aarch64-linux-android-addr2line"
+
+# Run the binary size script from the buildroot directory so that the treemap
+# path navigation will start from there.
+cd $ENGINE_BUILDROOT
```

Comment:
Does this need to be "$ENGINE_BUILDROOT" to guard against spaces in the path? Same question about the usages of $INPUT_PATH and other variables below.

Reply:
done